data/gnome/logitech-g703.svg

```diff
@@ -0,0 +1 @@
+logitech-g403.svg
```

Comment:
Looks like this is not a SVG :)

Reply:
This is a symlink to logitech-g403.svg.

Reply:
Right, I have been mistaken by github's web ui.

In that case, wouldn't it be better to just have `logitech-g403.svg` in the `.device` file and add a comment saying both are the same. Having a separate file is just misleading IMO.

Reply:
There is at least one other symlink that follows the current approach, as far as I know.